runtime/browser/web_view.h

```diff
@@ -87,6 +88,11 @@ class WebView {
         WebView* /*view*/,
         const std::string& /*url*/,
         std::function<void(bool)> /*result_handler*/) {}
+#ifdef PROFILE_WEARABLE
+    virtual void OnRotaryEvent(
+        WebView* /*view*/,
+        Eext_Rotary_Event_Info* info) {}
```

Comment:
Eext_Rotary_Event_Info
Change to newly defined value. 
To remove Dependency of EFL-Extension